### PR TITLE
Add documentation for JLR InControl component

### PR DESCRIPTION
**Pull request by rb28z2**
**Description:**
Add document for the Jaguar/Land Rover InControl component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24211

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by HarryJHolmes — COMMENTED** [review]
So is ‘name’ required it suggests here, it is  not?

**Review by HarryJHolmes — DISMISSED** [review]

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown]:
```suggestion
ha_iot_class: Cloud Polling
```

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown]:
```suggestion
ha_release: 0.95
```

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown]: Please remove the `redirect_from` section

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown:31]: Only required items in the first example

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown]:
```suggestion
  password: 'PASSWORD'
```

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown]:
```suggestion
  username: 'EMAIL_ADDRESS'
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown:11]:
```suggestion
ha_category:
  - Car
  - Binary Sensor
  - Sensor
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by rb28z2** [comment]
> So is ‘name’ required it suggests here, it is not?

Correct, `name` is required.

**Comment by HarryJHolmes** [comment]
Are the changes requested preventing merge? @rb28z2

**Comment by rb28z2** [comment]
I believe I addressed those. Just needs a re-review :)

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown]:
```suggestion
ha_release: 0.96
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by HarryJHolmes** [comment]
@rb28z2

**Comment by rb28z2** [comment]
@klaasnicolaas I've bumped the released version - hopefully its good now.

File: source/_components/jlrincontrol.markdown
Comment (klaasnicolaas) [inline source/_components/jlrincontrol.markdown]:
```suggestion
  required: true
```

name was required right?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by rb28z2** [comment]
I addressed the name field issue. Slip on my part 😅

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
✅ Can be merged after merging the parent PR 👍

**Pull request closed without merging**
